src/component/books/Book.js

```diff
@@ -1,30 +1,44 @@
-// eslint-disable-next-line no-unused-vars
-import React, { useEffect } from 'react';
+import React from 'react';
+import { AiOutlineLoading3Quarters } from 'react-icons/ai';
 import { useSelector, useDispatch } from 'react-redux';
-import { removeBook, fetchBooks } from '../../Redux/actions/BookAction';
+import { removeBook } from '../../Redux/actions/BookAction';
 
 const Book = () => {
   const book = useSelector((state) => state.BooksReducer);
   const dispatch = useDispatch();
   const { Books } = book;
 
-  useEffect(() => {
-    dispatch(fetchBooks());
-  }, []);
-
   if (Books.length === 0) {
     return <h1>Loading...</h1>;
   }
   return (
-    <div>
+    <div className="books">
       {
         Books.map((item) => {
           const { id, title, category } = item;
           return (
-            <div key={id}>
-              <h2>{title}</h2>
-              <h3>{category}</h3>
-              <button type="button" onClick={() => dispatch(removeBook(id))}>Remove Book</button>
+            <div key={id} className="book-item">
+              <div className="items">
+                <h3>{category}</h3>
+                <h2>{title}</h2>
+                <div className="buttons">
+                  <button type="button">comments</button>
+                  <button type="button" onClick={() => dispatch(removeBook(id))}>Remove Book</button>
+                  <button type="button">Edit</button>
+                </div>
+              </div>
+              <div className="percent">
+                <AiOutlineLoading3Quarters className="meter" />
+                <h3>
+                  <span className="percentage"> 64%</span>
+                  <span className="completed">completed</span>
+                </h3>
+              </div>
+              <div className="chapter">
+                <h4>Current Chapter</h4>
+                <h3>chapter one: &quot; A Lesson Learned &quot;</h3>
+                <button type="button">Update progress</button>
+              </div>
```

Comment:
- [ ] Each `book` is missing the vertical separators `|` between `progress` and `Chapter info`. Also, the white space between them is a bit large, Kindly update your `book` component to match the Zeplin design.
Your styling
![image](https://user-images.githubusercontent.com/33767951/141312020-3de64845-3ad7-4fd1-9ace-8ee13502e9da.png)
Zeplin design
![image](https://user-images.githubusercontent.com/33767951/141312060-725c5507-06cd-4ab9-becc-961e4357b83a.png)

- [ ] Each `book` is missing the `Author` name. Also, the vertical separators `|` between `Comment`, `Remove`, and `Edit` are missing, Kindly update your `book` component to match the Zeplin design(text colors are not matching).
Your styling
![image](https://user-images.githubusercontent.com/33767951/141311203-558d1756-992f-419d-b130-91854ece7c92.png)
Zeplin design
![image](https://user-images.githubusercontent.com/33767951/141311234-629b724e-69cc-438a-8d31-164a8ed01003.png)